src/store/account/account.store.impl.ts

```diff
@@ -0,0 +1,81 @@
+import { action, makeObservable, observable } from 'mobx';
+import { AccountStore } from './account.store';
+import { StoreImpl } from '../store/store.impl';
+import { User } from '../../models';
+
+const LOCAL_STORAGE_KEYS = {
+  token: 'token',
+  userId: 'user_id',
+};
+
+export class AccountStoreImpl extends StoreImpl implements AccountStore {
+  private token?: string;
+  private userId?: string;
+  private user?: User;
+
+  constructor() {
+    super();
+    this.loadToken();
+    this.loadUserId();
+
+    this.user = undefined;
+
+    makeObservable<AccountStoreImpl, any>(this, {
+      user: observable,
+      resetStore: action,
+      loadToken: action,
+      loadUserId: action,
+      setUser: action,
+      userExists: action,
+    });
+  }
+
+  async resetStore() {
+    this.token = '';
+    this.userId = '';
+    this.user = undefined;
+    await localStorage.removeItem(LOCAL_STORAGE_KEYS.token);
```

Comment:
Create a local storage service and add two functions. `save` and `delete`. Also add keys as static in that service.

Reply:
Call StorageService.add(StorageService.KEYS.token, 'token')